pkg/network/ebpf/c/prebuilt/http.c

```diff
@@ -606,9 +668,25 @@ int uprobe__crypto_tls_Conn_Read__return(struct pt_regs *ctx) {
     }
 
     // Read the TGID and goroutine ID to make the partial call key
-    go_tls_read_args_key_t call_key = {0};
+    go_tls_function_args_key_t call_key = {0};
     call_key.pid = pid;
 
+    uint64_t bytes_read = 0;
+    if (read_location(ctx, &pd->read_return_bytes, sizeof(bytes_read), &bytes_read)) {
+        log_debug("[go-tls-read-return] failed reading read return bytes location for pid %d\n", pid);
```

Comment:
```suggestion
        log_debug("[go-tls-read-return] failed reading read-return-bytes location for pid %d\n", pid);
```

Reply:
I just removed the "read" since the [] says which function we are talking about